karma.conf.cjs

```diff
@@ -50,7 +50,7 @@ module.exports = function (config) {
             type: "asset/resource",
           },
           {
-            test: /\.(js|tsx?)$/,
+            test: /\.(js|ts|tsx)$/,
```

Comment:
Both regex as identical as the `?` operator means `x` can be present `0` or `1` times. Or did this change fix anything?

Reply:
There are problems with import of `ehters` in karma environment so I was just trying things out.